src/ts/components/transaction/Send.tsx

```diff
@@ -150,17 +150,22 @@ class Send extends React.Component<ISendProps, ISendState> {
       .transfer(this.state.toAddress, BnAmount)
 
     const signOptions: SignerOptions = {
-      blockNumber: (await this.api.query.system.number()) as unknown as BN,
-      blockHash: await this.api.genesisHash,
-      genesisHash: await this.api.genesisHash,
+      blockNumber: await this.api.query.system.number() as unknown as BN,
+      blockHash: this.api.genesisHash,
+      genesisHash: this.api.genesisHash,
       nonce: await this.api.query.system.accountNonce(currentAddress) as Index
     }
-
+    const payloadValue: ExtrinsicPayloadValue = {
+      era: extrinsic.era,
+      method: extrinsic.method.toHex(),
+      nonce: signOptions.nonce,
+      tip: 0
```

Comment:
always 0? is it default to 0?

Reply:
Not always 0. It's a new field in the coming Kusama chain Extrinsic. Since we don't populate it yet, set it to 0 for now